floodviz/static/js/floodviz_onready.js

```diff
@@ -17,12 +17,47 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	FV.map_figure.init();
 
 	
-	// Hydrograph 
+	/* Hydrograph Setup */
+
+	// map interactions for hydrograph events
+	var maptip = d3.select('#maptip');
+	/**
+	 	* Shows/removes sitename tooltip on map figure at correct location.
+	 */
+	var show_map_tooltip = function(sitename, sitekey) {
+		var gage_point_cords = document.getElementById('map'+sitekey).getBoundingClientRect();
+		maptip.transition().duration(500);
+		maptip.style("display", "inline-block")
+			.style("left", (gage_point_cords.left) + 7 + "px")
+			.style("top", (gage_point_cords.top - 45) + "px")
+			.html((sitename));
+	}
```

Comment:
with removal of the global scope, show_map_tooltip and remove_map_tooltip code had to be duplicated into this file. I would like to not remove them from map.js because mapmoudle is using these functions itself to work its own tooltip hover. Any ideas on how to avoid duplicating this code?